tests/fail/parsingBoth/chars cannot be outside a subset of the ascii charset2.mm0

```diff
@@ -0,0 +1,6 @@
+-- lexeme ::= symbol | identifier | number | math-string
+-- symbol ::= '*' | '.' | ':' | ';' | '(' | ')' | '>' | '{' | '}' | '=' | '_'
+-- identifier ::= [a-zA-Z_][a-zA-Z0-9_]*
+-- number ::= 0 | [1-9][0-9]*
+-- math-string ::= '$' [^$]* '$'
+sort s ;
```

Comment:
I see a space between the `s` and `;`. The comment should say something like:

```
--    v-- this is a nonbreaking space
sort s ;
```
if the intent is to see the effect of non-ascii characters. (But the character I've pointed at is just a plain space so I'm not sure what the test actually is.)